files/scripts/supervisor-proc-exit-listener

```diff
@@ -118,7 +120,7 @@ def main(argv):
 
     critical_group_list, critical_process_list = get_critical_group_and_process_list()
 
-    process_under_alerting = {}
+    process_under_alerting = defaultdict(list)
```

Comment:
List is not the best datatype here. each process will always contain two fields, the time of last alert and the number of times it has alerted. This is better suited for a dict or named tuple.

Reply:
Great suggestion! Fixed.